scripts/update/01-run.sh

```diff
@@ -54,6 +54,14 @@ if [[ ! -z "${UMBREL_OS:-}" ]]; then
         echo "ERROR: No Umbrel installation found at SD root ${SD_CARD_UMBREL_ROOT}"
         echo "Skipping updating on SD Card..."
     fi
+    
+    # Configure unattended-upgrades
+    # This can be safely removed from Umbrel v0.3.x+
+    echo "Configuring automated updates of apt packages"
+    apt-get update
+    DEBIAN_FRONTEND=noninteractive apt-get install unattended-upgrades -y
+    echo unattended-upgrades unattended-upgrades/enable_auto_updates boolean true | debconf-set-selections
```

Comment:
So this will run in the background automatically, right?

I don't think we should do that since it's essentially a back door. This is the reason we don't automatically update Umbrel already and require the user to confirm first.

We should do security updates only during OTA so users who don't want to opt in to trusting third parties don't have software automatically installed on their hardware.

We should run `unattended-upgrade` once during OTA and ensure it isn't running automatically in the background.

Reply:
But why can't we just use apt then? I think unattended-updates was made to be used automatically.

Reply:
Because `unattended-upgrade` only updates security critical dependencies.

Reply:
`apt-get -s dist-upgrade | grep "^Inst" | grep -i securi | awk -F " " {'print $2'} | xargs apt-get install`

That should work too, and isn't an additional dependency

Reply:
Wouldn't this have the potential to match packages with the string "securi" as part of their name?

I think it would be more reliable to use `unattended-upgrade` for this which is purpose built for this use case.

Reply:
Okay, I'll do that